Please do not attempt to install when running bare make

```
user@debian:~/AdaptiveCpp/build$ make -j14
...
[ 93%] Building CXX object src/runtime/CMakeFiles/rt-backend-omp.dir/omp/omp_phys_mem.cpp.o
[ 94%] Building CXX object src/runtime/CMakeFiles/rt-backend-omp.dir/omp/omp_code_object.cpp.o
[ 94%] Linking CXX executable acpp-info
[ 94%] Built target acpp-info
[ 94%] Linking CXX shared library librt-backend-omp.so
[ 94%] Built target rt-backend-omp
[ 95%] Linking CXX static library libLLVMSPIRVLib.a
[ 95%] Built target LLVMSPIRVLib
[ 97%] Building CXX object tools/llvm-spirv/CMakeFiles/llvm-spirv.dir/llvm-spirv.cpp.o
[100%] Linking CXX executable llvm-spirv
[ 94%] Linking CXX shared module libacpp-clang.so
[ 94%] Built target acpp-clang
[100%] Built target llvm-spirv
Install the project...
-- Install configuration: ""
CMake Error at cmake_install.cmake:54 (file):
  file cannot create directory:
  /usr/local/lib/hipSYCL/ext/llvm-spirv/include/LLVMSPIRVLib.  Maybe need
  administrative privileges.


gmake[3]: *** [Makefile:100: install] Error 1
make[2]: *** [src/compiler/llvm-to-backend/CMakeFiles/LLVMSpirvTranslator.dir/build.make:86: src/compiler/llvm-to-backend/LLVMSpirvTranslator-prefix/src/LLVMSpirvTranslator-stamp/LLVMSpirvTranslator-build] Error 2
make[1]: *** [CMakeFiles/Makefile2:967: src/compiler/llvm-to-backend/CMakeFiles/LLVMSpirvTranslator.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
user@debian:~/AdaptiveCpp/build$ 
```

cmake version 3.31.6

--- a/src/compiler/llvm-to-backend/CMakeLists.txt
+++ b/src/compiler/llvm-to-backend/CMakeLists.txt
@@ -196,8 +196,17 @@ if(WITH_SSCP_COMPILER)
         -DLLVM_DIR:PATH=${LLVM_DIR}
         -DCMAKE_INSTALL_PREFIX:PATH=${LLVMSPIRV_INSTALLDIR}
         -DCMAKE_CXX_FLAGS:STRING=${CMAKE_CXX_FLAGS}
-      BUILD_COMMAND ${CMAKE_COMMAND} --build <BINARY_DIR> --config Release --target install
+      BUILD_COMMAND ${CMAKE_COMMAND} --build <BINARY_DIR> --config Release
+      INSTALL_COMMAND ""
     )
+
+    add_custom_target(InstallLLVMSpirvTranslator
+      COMMAND ${CMAKE_COMMAND} --build ${CMAKE_CURRENT_BINARY_DIR}/ext/llvm-spirv --target install
+      WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/ext/llvm-spirv
+    )
+
+    #install(TARGETS InstallLLVMSpirvTranslator)
+    install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} --build . --target InstallLLVMSpirvTranslator)")
 
     add_dependencies(llvm-to-spirv LLVMSpirvTranslator)
     target_compile_definitions(llvm-to-spirv PRIVATE

